app/src/main/java/com/sunasterisk/a14day_challenge/ui/login/LoginContract.kt

```diff
@@ -4,13 +4,13 @@ import com.sunasterisk.a14day_challenge.data.UserRepository
 import com.sunasterisk.a14day_challenge.data.model.User
 
 interface LoginContract {
-    interface View{
+    interface View {
         fun onLoginSucceeded(user: User)
 
         fun onLoginFailure(error: String)
     }
```

Comment:
Đặt tên hàm mô tả hành động nội dung bên trong hàm thay vì mục đích nó được gọi 
```suggestion
    interface View {
        fun changeToHomeScreen(user: User)

        fun showError(error: String)
    }
```